test/image-diff_test.js

```diff
@@ -45,12 +45,51 @@ describe('image-diff', function () {
     });
   });
 
+  describe('diffing different images using lower threshold', function () {
+    runImageDiff({
+      actualImage: __dirname + '/test-files/checkerboard.png',
+      expectedImage: __dirname + '/test-files/white.png',
+      diffImage: __dirname + '/actual-files/different.png',
+      threshold: 0.7
```

Comment:
Can we make this something like `0.2` to make it a very obvious threshold difference to be over?